Actively/Actively/ActivelyApp/Controllers/GameController.cs

```diff
@@ -0,0 +1,145 @@
+﻿using ActivelyApp.CustomExceptions;
+using ActivelyApp.Models.Common;
+using ActivelyApp.Models.EntityDto;
+using ActivelyApp.Services.EntityService;
+using ActivelyDomain.Entities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Resources;
+using System.Numerics;
+
+namespace ActivelyApp.Controllers
+{
+    [ApiController]
+    [Authorize]
+    [Route("[controller]/[action]")]
+    public class GameController : Controller
+    {
+        private readonly IGameService _gameService;
+
+        public GameController(IGameService gameService)
+        {
+            _gameService = gameService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetAll()
+        {
+            IEnumerable<GameDto> games = new List<GameDto>();
```

Comment:
Why do we initialize List<GameDto> in line 28 if we override it in line 32. I think we should declare a "games" variable here